libs9s/s9srpcclient.cpp

```diff
@@ -3801,7 +3801,7 @@ S9sRpcClient::registerHost()
         command = "registernode";
         title   = "Register Mongos Node";
     } else {
-        command = protocol;
+        command = protocol.empty() ? "registernode" : protocol;
```

Comment:
I think the protocol must not be empty. That we way could not know what is the node type to register, so we could not know what runnign process to look for, what port to try to connect too when trying to collect node info.

Reply:
Thinking about it a bit more, we might have some logic in cmon to assume the node type based on cluster type.
May be this is as well can go in.